### PR TITLE
fix: hack to fix the checking of `remote_addrs`

### PR DESCRIPTION
**Pull request by nic-chen**
Please answer these questions before submitting a pull request

- Why submit this pull request?
- [x] Bugfix
- [ ] New feature provided
- [ ] Improve performance

- Related issues

related #940 

we should not sync the json schema outside APISIX v2.1, so we need to hack to fix it.

### CODE REVIEW

File: api/internal/core/store/validate.go [hunk 1]
```diff
@@ -201,6 +210,9 @@ func checkConf(reqBody interface{}) error {
 		if err := checkUpstream(route.Upstream); err != nil {
 			return err
 		}
+		if err := checkRemoteAddr(route.RemoteAddrs); err != nil {
```
Comment (membphis) [inline api/internal/core/store/validate.go:213]: need a todo comment: this is a temporary method, we'll drop it later

File: api/internal/core/store/validate_test.go [hunk 2]
```diff
@@ -247,5 +247,132 @@ func TestAPISIXJsonSchemaValidator_checkUpstream(t *testing.T) {
 	err = validator.Validate(route5)
 	assert.NotNil(t, err)
 	assert.EqualError(t, err, "schema validate failed: (root): Does not match pattern '^((uri|server_name|server_addr|request_uri|remote_port|remote_addr|query_string|host|hostname)|arg_[0-9a-zA-z_-]+)$'")
+}
 
+func TestAPISIXJsonSchemaValidator_Route_checkRemoteAddr(t *testing.T) {
```
Comment (membphis) [inline api/internal/core/store/validate_test.go:252]: test case: `"remote_addr": ""`

**Review by membphis — APPROVED** [review]

**Comment by membphis** [comment]
@gxthrj @moonming @ShiningRush please review this PR if you have time ^_^

File: api/internal/core/store/validate_test.go [hunk 3]
```diff
@@ -247,5 +247,134 @@ func TestAPISIXJsonSchemaValidator_checkUpstream(t *testing.T) {
...
+					"nodes": [{
+						"host": "127.0.0.1",
+						"port": 8080,
+						"weight": 1
+					}],
+					"type": "roundrobin"
+				},
+				"remote_addrs": ["127.0.0.1", "192.0.0.0/8", "::1", "fe80::1/64"]
+			}`,
+		},
```
Comment (gxthrj) [inline api/internal/core/store/validate_test.go:323]: There is no assert, is it default ok?

Comment (nic-chen) [reply api/internal/core/store/validate_test.go:323]: yes, it is. if it cause err, test will fail.

**Review by gxthrj — APPROVED** [review]

**Pull request merged**